Fix lang key errors

jhinkey opened the pull request:
/cc @codyhoag

codyhoag committed:
LRDOCS-1158 Fix missing lang key errors

Commit:
LRDOCS-1158 Regenerate lang keys

Commit:
LRDOCS-1158 Update JSP to use new language key name

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 48 minutes 36 seconds</p><h4>Job Results:</h4><p>15 Jobs Passed.<br />1 Job Failed.</p><h1>See the following links for more information.</h1><ul><li><h4><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/848/">test-portal-acceptance-pullrequest(master)</a></h4><pre>Completed with the status of FAILURE.</pre></li><li><h4><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3628/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/portal-smoke</a></h4></li><li><h4><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3636/">test-portal-acceptance-pullrequest-batch(master)/integration-db2</a></h4></li><li><h4><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3665/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql</a></h4></li><li><h4><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3686/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle</a></h4></li><li><h4><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3677/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql</a></h4></li><li><h4><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3625/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2</a></h4></li><li><h4><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3585/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql</a></h4></li><li><h4><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3647/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle</a></h4></li><li><h4><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3629/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql</a></h4><h5>Job Results:</h5><p>3573 Tests Passed.<br />1 Test Failed.</p><ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=3,label_exp=!master/3629//testReport/com.liferay.document.library.search.test/DLFileEntrySearchTest/testSearchTikaRawMetadata">DLFileEntrySearchTest.testSearchTikaRawMetadata</a></li></ul></li><li><h4><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3637/">test-portal-acceptance-pullrequest-batch(master)/modules-unit</a></h4></li><li><h4><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3666/">test-portal-acceptance-pullrequest-batch(master)/plugins</a></h4></li><li><h4><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3687/">test-portal-acceptance-pullrequest-batch(master)/portal-web</a></h4></li><li><h4><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3678/">test-portal-acceptance-pullrequest-batch(master)/service-builder</a></h4></li><li><h4><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3626/">test-portal-acceptance-pullrequest-batch(master)/tck</a></h4></li><li><h4><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/3586/">test-portal-acceptance-pullrequest-batch(master)/unit</a></h4></li><li><h4><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/908/">test-portal-acceptance-pullrequest-source(master)</a></h4></li></ul>

brianchandotcom commented:
@jhinkey can you resend? I fixed most of these myself already, but I did it the opposite way. I fixed the JSP because I wasn't sure which one was right.

Can you rollback my changes and resend the proper fix? Thx. (There are also rebase errors because of what I did)

See 0179740d21306b60634e944317d7beb8f6926d47 and 9a02bef08c48bc6aa2a48464e44b81b41b33dc27

jhinkey commented:
Yep, I'll definitely do that. Thanks, Brian!

On 8/5/2015 1:23 AM, Brian Chan wrote:
>
> @jhinkey <https://github.com/jhinkey> can you resend? I fixed most of
> these myself already, but I did it the opposite way. I fixed the JSP
> because I wasn't sure which one was right.
>
> Can you rollback my changes and resend the proper fix? Thx. (There are
> also rebase errors because of what I did)
>
> See 0179740
> <https://github.com/brianchandotcom/liferay-portal/commit/0179740d21306b60634e944317d7beb8f6926d47>
> and 9a02bef
> <https://github.com/brianchandotcom/liferay-portal/commit/9a02bef08c48bc6aa2a48464e44b81b41b33dc27>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brianchandotcom/liferay-portal/pull/28950#issuecomment-127861875>.
>

-- 
Jim Hinkey
Sr. Knowledge Engineer
Liferay, Inc.
www.liferay.com <http://www.liferay.com>

*Liferay Symposium North America*
November 16-17, 2015 | Chicago, IL
www.liferay.com/northamerica2015 <http://www.liferay.com/northamerica2015>
---

Visit Us: www.liferay.com <http://www.liferay.com> | Like Us:
facebook.com/liferay <http://facebook.com/liferay> | Follow Us:
twitter.com/liferay <http://twitter.com/liferay>

jhinkey commented:
Sent https://github.com/brianchandotcom/liferay-portal/pull/28992
